test/snapshots/CodegenMonad.purs

```diff
@@ -50,6 +50,8 @@ test = unsafePartial do
           [ exprIdent "a"
           , exprIdent "b"
           ]
+      write $ declSignature "holy" $ typeHole "?SomeHole"
+      write $ declValue "holy" [] $ exprHole "?someHole"
```

Comment:
Is this testing something that the examples don't already test?

Reply:
Uh, no... :grin: